operations/system_admin_toolkit/SAT_Uninstall_and_Downgrade.md

```diff
@@ -67,29 +67,28 @@ releases are the last releases of SAT as a separate product stream.
 
 ## Downgrade: Switch Between SAT Versions
 
-Starting in CSM v1.6.0, it is no longer recommended to use `prodmgr activate` to switch between SAT
+Starting in CSM 1.6.0, it is no longer recommended to use `prodmgr activate` to switch between SAT
 versions.
 
 Instead, if it is necessary to switch to an alternate version of SAT, it is recommended to set the
 environment variable `SAT_IMAGE` as described in the procedure below. This will change the version
 of the `cray-sat` container image run by Podman when the `sat` command is executed.
 
-### Downgrade Procedure
+1. (`ncn-mw#`) Determine the versions of the `cray-sat` container image which are available
+   in the container image registry in Nexus.
 
-1. (`ncn-mw#`) First, determine the versions of the `cray-sat` container image which are available
-   in the container image registry in Nexus. There are multiple ways to do so. One easy way is to
-   use `podman search`.
+   There are multiple ways to do this, but one easy way is to use `podman search` because:
```

Comment:
I don't think "because" is actually appropriate here. These aren't reasons why we use `podman search` as opposed to any other tool for viewing the contents of the image registry in Nexus. They're just additional info about where the admin should expect to see the image based on the version(s) of CSM and SAT installed.

Reply:
Thank you! That wasn't entirely clear to me. Updating to the following wording in the next commit:

```suggestion
   There are multiple ways to do this, but one easy way is to use `podman search` to find
   the `cray-sat` container image within the following locations:
```